### PR TITLE
Add in support for AWS session tokens

### PR DESCRIPTION
**Pull request by alexlatchford**
Great little script! We have a need to grab session tokens (for federated logins) so I've added support let me know what you think

### CODE REVIEW

**Review by whereisaaron — CHANGES_REQUESTED** [review]
Looks good! You just need to update this documentation to mention the new option

echo "  You can specify one --key or --secret to get just that value, with no line break,"

File: get-aws-profile.sh [hunk 1]
```diff
@@ -43,14 +43,15 @@ cfg_parser ()
 
 display_usage ()
 {
-  echo "Usage: $0 [--credentials=<path>] [--profile=<name>] [--key|--secret]"
+  echo "Usage: $0 [--credentials=<path>] [--profile=<name>] [--key|--secret|--session-token]"
   echo "  Default --credentials is '~/.aws/credentials'"
   echo "  Default --profile is 'default'"
   echo "  By default environment variables are generate, e.g."
   echo "    source \$($0 --profile=myprofile)"
   echo "  You can specify one --key or --secret to get just that value, with no line break,"
```
Comment (whereisaaron) [inline get-aws-profile.sh]: Update this comment for the new option and we are good to go

**Comment by mkj28** [comment]
@alexlatchford  plz :) update/let's get it in :)

**Comment by whereisaaron** [comment]
Hi @mkj28 @alexlatchford, I updated the usage comment myself and merged it.

**Comment by mkj28** [comment]
awesome, thanks!

**Pull request merged**